### ISSUE TITLE
Support Django 1.8 & 1.9

### ISSUE THREAD

**Issue by cschneid** [scoutapp/scout_apm_python#26]

**Comment by sebzur** [comment]
@cschneid Any estimated deadline for this issue?

**Comment by mtthwsar** [comment]
This would be useful for me too!

**Comment by ckcollab** [comment]
Thanks for all of the hard work on this! I was bummed to get through setup and realize Django 2.0 isn't supported; hope this works soon.

**Comment by itsderek23** [comment]
Hi @ckcollab - odd. Scout scout support Django 2.0.

Mind emailing support@scoutapp.com with any errors you are seeing?

**Comment by ckcollab** [comment]
Ah, misread the readme, hopefully it's a simple mistake on my end! Here's the error:

```python
       Traceback (most recent call last):
         File "manage.py", line 15, in <module>
           execute_from_command_line(sys.argv)
         File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
           utility.execute()
         File "/app/.heroku/python/lib/python3.6/site-packages/django/core/management/__init__.py", line 347, in execute
           django.setup()
         File "/app/.heroku/python/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
           apps.populate(settings.INSTALLED_APPS)
         File "/app/.heroku/python/lib/python3.6/site-packages/django/apps/registry.py", line 120, in populate
           app_config.ready()
         File "/app/.heroku/python/lib/python3.6/site-packages/scout_apm/django/apps.py", line 30, in ready
           self.install_middleware()
         File "/app/.heroku/python/lib/python3.6/site-packages/scout_apm/django/apps.py", line 44, in install_middleware
           if settings.MIDDLEWARE_CLASSES is not None:
         File "/app/.heroku/python/lib/python3.6/site-packages/django/conf/__init__.py", line 57, in __getattr__
           val = getattr(self._wrapped, name)
       AttributeError: 'Settings' object has no attribute 'MIDDLEWARE_CLASSES'
```

I can open a new issue for this?

**Comment by cschneid** [comment]
@ckcollab I'll open an issue and get that fixed. Sorry!

**Comment by cschneid** [comment]
@ckcollab the issue is #118, and I just pushed a PR for it to #119 

It'll be released in a little bit as version 1.3.1

**Comment by ajford** [comment]
Also would love to see Django 1.8/1.9 support. 

Django 1.8 on Ubuntu 16.04 running behind uwsgi, if details help.

**Comment by aaugustin** [comment]
Once #133 is merged, I think this issue can be closed.
